content/topics/web-frontend/react/unit-testing/_index.md

```diff
@@ -6,4 +6,7 @@ title: React Unit testing
 ---
 
 - https://www.youtube.com/watch?v=-bmdf1oATQo
-- https://www.toptal.com/react/tdd-react-user-stories-to-development
\ No newline at end of file
+- https://www.toptal.com/react/tdd-react-user-stories-to-development
+- https://testing-library.com/docs/react-testing-library/intro/
+- https://jestjs.io/
+- https://www.youtube.com/watch?v=3e1GHCA3GP0
```

Comment:
Please update the following links to markdown to make it more consistent with other links in the syllabus